patches/core/ungoogled-chromium/fix-building-without-safebrowsing.patch

```diff
@@ -1123,15 +1157,15 @@
  
  #if defined(OS_ANDROID)
  #include "chrome/browser/android/tab_android.h"
-@@ -828,6 +827,7 @@ autofill::LanguageCode ChromePasswordMan
+@@ -830,6 +829,7 @@ autofill::LanguageCode ChromePasswordMan
    return autofill::LanguageCode();
  }
  
 +#if BUILDFLAG(FULL_SAFE_BROWSING)
  safe_browsing::PasswordProtectionService*
  ChromePasswordManagerClient::GetPasswordProtectionService() const {
    return safe_browsing::ChromePasswordProtectionService::
-@@ -838,13 +838,6 @@ ChromePasswordManagerClient::GetPassword
+@@ -840,13 +840,6 @@ ChromePasswordManagerClient::GetPassword
  void ChromePasswordManagerClient::CheckSafeBrowsingReputation(
      const GURL& form_action,
      const GURL& frame_url) {
```

Comment:
The only thing that I noticed is that the code that is being removed between here and the next `#endif // BUILDFLAG(FULL_SAFE_BROWSING)` does not need to be removed, since it is ignored at the pre-processor stage. This is a very minor issue; if it isn't resolved when I next port this patch to a new version of Chromium, I will address it then.